service/src/main/resources/logback-spring.xml

```diff
@@ -117,17 +130,20 @@
 	<logger name="ErrorLogger" level="trace" additivity="false">
 		<appender-ref ref="errorLog"/>
 		<appender-ref ref="console"/>
+		<appender-ref ref="consoleToFileLog"/>
 	</logger>
 	<logger name="PrivilegeLogger" level="trace" additivity="false">
 		<appender-ref ref="privilegeLog"/>
```

Comment:
Gibt es einen Grund warum consoleToFileLog nicht im PrivilegeLogger ist?

Reply:
Ich habe es nur da hinzugefügt, wo vorher schon `<appender-ref ref="console"/>` stand. So wie ich das verstehe erscheint PrivilegeLogger (und auch ein paar andere) also nicht in der Konsole, deshalb erwarte ich sie auch nicht im `consoleToFileLog`.
Habe aber keine starke Meinung, wenn ihr meint es sollte dazu schreibe ich es noch mit rein.